tests/test_integration.py

```diff
@@ -167,6 +171,7 @@ def test_element_str_from_highavailability_with_ha1_and_ha2_children(self):
     # 2) VirtualRouter with StaticRoute child
     def test_element_str_from_virtualrouter_with_sr_parent(self):
         '''StaticRoute > VirtualRouter'''
+        self.maxDiff = None
```

Comment:
When I was debugging the issue with the strings splitting up, I needed to see the full expected and return value not the clipped versions. I can delete that line if needed.